gateway/versions_handler.go

```diff
@@ -76,22 +74,25 @@ func (h *VersionsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			continue
 		}
 
-		meta = VersionMeta{
+		versionMetas.Metas = append(versionMetas.Metas, VersionMeta{
 			ID:               id,
 			Name:             currentAPI.Name,
 			VersionName:      name,
 			Internal:         currentAPI.Internal,
 			ExpirationDate:   currentAPI.Expiration,
 			IsDefaultVersion: baseAPI.VersionDefinition.Default == name,
-		}
-
-		versionMetas.Metas = append(versionMetas.Metas, meta)
+		})
 	}
 
 	sort.Slice(versionMetas.Metas, func(i, j int) bool {
 		return versionMetas.Metas[i].VersionName < versionMetas.Metas[j].VersionName
 	})
 
+	if baseAPIMeta.ID != "" {
```

Comment:
This `if` is redundant, I'm asuming we're going to error out at L46

Reply:
baseAPIMeta is already declared, so appending it without a check would add an entry with empty values